web_quick_start_screen/wizards/quick_start_screen.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import api, models
+
+
+class QuicktStartScreen(models.TransientModel):
+    _name = "quick.start.screen"
+    _description = "Quick action selection screen"
+
+    @api.model
+    def _get_start_screen_actions(self):
+        """Extend to add more options to the start screen"""
+        actions = []
+        screen_actions = self.env["quick.start.screen.action"].search([])
+        for action in screen_actions:
+            actions.append(
+                {
+                    "title": action.name,
+                    "description": (action.description),
+                    "image": action.image,
+                    "icon": (
+                        f"fa {action.icon_name or 'fa-flash'} "
+                        f"o_start_icon_color_{action.color}"
+                    ),
+                    "action_name": action.action_ref_id.id,
```

Comment:
Warning the field is non mandatory and will break with the following stacktrace
```
Traceback (most recent call last):
  File "/opt/odoo/odoo/http.py", line 1649, in _serve_db
    return service_model.retrying(self._serve_ir_http, self.env)
  File "/opt/odoo/odoo/service/model.py", line 133, in retrying
    result = func()
  File "/opt/odoo/odoo/http.py", line 1676, in _serve_ir_http
    response = self.dispatcher.dispatch(rule.endpoint, args)
  File "/opt/odoo/odoo/http.py", line 1880, in dispatch
    result = self.request.registry['ir.http']._dispatch(endpoint)
  File "/opt/odoo/odoo/addons/base/models/ir_http.py", line 154, in _dispatch
    result = endpoint(**request.params)
  File "/opt/odoo/odoo/http.py", line 732, in route_wrapper
    result = endpoint(self, *args, **params_ok)
  File "/mnt/data/odoo-addons-dir/web_quick_start_screen/controllers/quick_start_screen.py", line 17, in quick_start_screen
    ]._get_start_screen_actions()
  File "/mnt/data/odoo-addons-dir/web_quick_start_screen/wizards/quick_start_screen.py", line 25, in _get_start_screen_actions
    "action_name": action.action_ref_id.id,
AttributeError: 'NoneType' object has no attribute 'id'
```

Reply:
Shouldn't this be an evalued expression?

Reply:
> Warning the field is non mandatory

Sure it should be. Thanks :)

> Shouldn't this be an evalued expression?

Not really, but the key name is a bit missleading, changing it

Reply:
OK, but we talked about some evalued thing (¿the context or domain?)

Reply:
OK, please indicate these hints in the README for others to be able to discover that usefulness.